projects/playground/e2e/integration/playground.spec.ts

```diff
@@ -170,7 +170,7 @@ describe('Smoke tests', () => {
 
   it('should not navigate to the lazy loaded module when not authenticated', () => {
     cy.visit('/lazy');
-    cy.get('[data-cy=lazy-module]').should('not.be.visible');
+    cy.get('[data-cy=lazy-module]').should('not.exist');
```

Comment:
All of the above tests started to fail because of using `not.be.visible`, it works fine when using `not.exists`.

Reply:
Presumably there was some change in behaviour in version of Cypress we upgraded to?

Reply:
Yes I think so as well, but I cant find anything in the changelog.